blablacar-persistence/src/test/java/cz/fi/muni/pa165/teamred/dao/RideDaoTest.java

```diff
@@ -114,29 +171,150 @@ public void testFindRideById(){
     }
 
     @Test
+    public void findRideByInvalidId(){
+        assertThat(rideDao.findById(2L)).isEqualTo(null);
+    }
+
+    @Test
+    void testFindRideByNullId() {
+        assertThatThrownBy(() -> rideDao.findById(null)).isInstanceOf(IllegalArgumentException.class);
+    }
+
+    //----------------------------------------------------------------------------------------------------Delete Related
+
+    @Test
     public void testDeleteRide(){
+        Ride validRide = new Ride();
+        initializeRide(validRide);
+        rideDao.create(validRide);
```

Comment:
vytvarat by sa mali cez entityManager v @BeforeMethod, aby si v teste pre RideDaoTest.delete() nevolal ziadnu inu RideDao metodu

Reply:
the same aj v metodach nizsie potom, pisal som ti aj koment na slacku, ked to persistnes cez entity managera v @BeforeMethod tak pri kazdej metode budes mat vstupne podmienky rovnake a vytvoreny ten pocet entit v DB co chces

Reply:
mne ten EM proste nešiel ale skusim ešte raz